unison-src/transcripts/mergeloop.md

```diff
@@ -0,0 +1,51 @@
+# Merge loop test
+
+This tests for regressions of https://github.com/unisonweb/unison/issues/1276 where trivial merges cause loops in the history.
+
```

Comment:
A trivial merge is one where the head values are the same?

Reply:
Yes.